History.markdown

```diff
@@ -1,6 +1,7 @@
 ## HEAD
 
   * Test against Jekyll 2 and 3. (#30)
+  * Lock to Listen 3.0.6
```

Comment:
oh- this is the pattern that I was looking for for Style Closet for 'things on deck'!